src/rubrix/client/sdk/client.py

```diff
@@ -72,6 +74,17 @@ def __del__(self):
     def __hash__(self):
         return hash(self.base_url)
 
+    def with_httpx_error_handler(func):
+        @functools.wraps(func)
+        def inner(self, *args, **kwargs):
+            try:
+                func(self, *args, **kwargs)
```

Comment:
The `func` result must be returned. Otherwise, the `func` won't work

Reply:
My bad. I should have been more thorough in testing it after the changes.